.gitignore

```diff
@@ -63,6 +63,8 @@ bindings/python/libmata.cpp
 bindings/python/test.dot
 bindings/python/test.dot.pdf
 bindings/python/MANIFEST
+bindings/python/tests/test.dot
```

Comment:
Probably it would be better to change to `*.dot` and `*.pdf`?

Reply:
Done.